geometry_msgs/Point[] left_lane
geometry_msgs/Point[] center_lane
geometry_msgs/Point[] right_lane
bool center_valid
builtin_interfaces/Time stamp
